README.md

```diff
@@ -395,8 +395,8 @@ Proxies enable creation of objects with the full range of behaviors available to
 // Proxying a normal object
 var target = {};
 var handler = {
-  get: function (receiver, name) {
-    return `Hello, ${name}!`;
+  get: function (target, property, receiver) {
+    return `Hello, ${property}!`;
```

Comment:
What about keeping "name", but change the parameters? Maybe use a different example that clarifies the use of this. For example:

```js
var person = { name: "Alice" };
var handlers = {
    get: function (target, property, receiver) {
        if (property === "realName") {
            return target.name;
        } else if (property === "fakeName") {
            return "Really " + receiver.name;
        }
        return "Bob";
    }
};

var fakePerson = new Proxy(person, handlers);
fakePerson.name === "Bob" &&
fakePerson.realName === "Alice" &&
fakePerson.fakeName === "Really Bob";
```